src/voice-pe/customizing/reassemble.md

```diff
@@ -8,11 +8,7 @@ zendesk:
 
 Follow these steps if you have 3D-printed your own enclosure parts and want to reassemble the device.
 
->Notice
->
->Risk of damage to equipment
->
->Disassembling and reassembling the device can void the warranty.
+{% hazard "notice" "Risk of damage to equipment Disassembling and reassembling the device can void the warranty." %}
```

Comment:
```suggestion
{% hazard "notice" "Risk of damage to equipment. Disassembling and reassembling the device can void the warranty." %}
```

Reply:
well, I was hoping to be able to have a title for the hazard statements

Reply:
https://github.com/NabuCasa/support/pull/498